jobs/gcp-discovery/src/main/java/com/tmobile/pacbot/gcp/inventory/collector/GKEClusterInventoryCollector.java

```diff
@@ -69,12 +69,7 @@ public List<GKEClusterVH> fetchGKEClusterInventory(ProjectVH project) throws IOE
                     gkeClusterVH.setProjectName(project.getProjectName());
                     gkeClusterVH.setProjectId(project.getProjectId());
                     gkeClusterVH.set_cloudType(InventoryConstants.CLOUD_TYPE_GCP);
-                    gkeClusterVH.setId(cluster.getId());
-                    gkeClusterVH.setProjectName(project.getProjectName());
-                    gkeClusterVH.setProjectId(project.getProjectId());
-                    gkeClusterVH.set_cloudType(InventoryConstants.CLOUD_TYPE_GCP);
-
-                    gkeClusterVH.setRegion(cluster.getLocation());
```

Comment:
Please confirm if this deletion is not accidental.

Reply:
there are not actually removed these line got added above line 69